stdlib/datetime.pyi

```diff
@@ -278,6 +279,7 @@ class datetime(date):
         def now(cls, tz: _TzInfo) -> datetime: ...
 
     @classmethod
+    @deprecated("Use timezone-aware objects to represent datetimes in UTC; e.g. by calling .now(datetime.UTC)")
```

Comment:
Not sure about this one. It may have always been discouraged to use this method, but I think the deprecation has been pretty disruptive — see https://discuss.python.org/t/deprecating-utcnow-and-utcfromtimestamp/26221/12?u=alexwaygood. For Python 3.8 users, the method is still going to be around for a while; do we need to bother them with this warning just yet? (On the other hand: if the method is widely used, perhaps it's all the more important to make sure the word gets out that it's deprecated...)

Reply:
I think it's worth it. Work had a bug just this week because someone was using these.

(Also I think it will be around for like eight years before CPython removes it, I'd definitely oppose removing it in 3.14)